CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - Minor: Update `CustomFieldInputNumber` to accept error and helpText props
```

Comment:
FYI: I think appending to the changelog is more friendly to possible future git merge conflicts.

Reply:
@cincospenguinos ^